cpu/cpu_openbsd.go

```diff
@@ -169,19 +168,10 @@ func InfoWithContext(ctx context.Context) ([]InfoStat, error) {
 	}
 	c.Mhz = float64(u32)
 
-	mib := []int32{CTLHw, NCpuOnline}
-	buf, _, err := common.CallSyscall(mib)
-	if err != nil {
-		return nil, err
-	}
-
-	var ncpu int32
-	br := bytes.NewReader(buf)
-	err = binary.Read(br, binary.LittleEndian, &ncpu)
-	if err != nil {
+	if u32, err = unix.SysctlUint32("hw.ncpuonline"); err != nil {
```

Comment:
could you please rename `u32` to `ncpu`?

Reply:
The idea was to reuse the `var` already declared and used to extract `hw.cpuspeed`, but I agree having a separate, properly named `var ncpu` makes it easier to read. FWIW, I also changed it above and renamed the existing `u32` to `mhz`.